src/forms/medication-dispense-review.component.tsx

```diff
@@ -489,6 +495,48 @@ const MedicationDispenseReview: React.FC<MedicationDispenseReviewProps> = ({
             });
           }}
         />
+
+        <DatePicker
+          datePickerType="single"
+          dateFormat={datePickerFormat}
+          minDate={dayjs(prescriptionDate).startOf('day').toDate()}
```

Comment:
What happens here if prescriptionDate = null?  That seems like it is possible from the earlier code

Reply:
Oh, interesting, I hadn't really thought of this, but since dayjs() returns the current datetime, I would think dayjs(null) would return the current datetime as well.

Prescription Date should never be null long-term, but as it is loaded via a hook, it might start null.  In this case, having the min date be today until the prescription is properly loaded like proper behavior?

But I also just might want to *not* display the form until all the data is loaded, I can look into that, as it might fix a weird issue I was seeing as well.

Reply:
> I would think dayjs(null) would return the current datetime as well.

I just checked. It actually creates an "Invalid Date" object if the input is null. It should be fine as long as we have a null check prior